activities/procurement/process.md

```diff
@@ -0,0 +1,12 @@
+---
+type: Resource
+---
+
+# Procurement process
+
+Our process for procurement of services, and in some cases goods, is as follows:
+
+1. we request a proposal or quote
+2. we check if price and value are in line with the market
+3. if products and services are in line with our [procurement principles](principles.md)
+4. the order is placed by the operations team and signed off by the chief executive and board of directors, if needed
```

Comment:
I have some style edit suggestions.

Numbered lists are usually full sentences and don't require lead-in sentences. The active tense is usually easier to read than the passive.

Proposal:

This is how we procure all services and some goods.

1. We request a proposal or quote.
2. We check if price and value are in line with the market.
3. We check that the proposed procurement meets our [procurement principles](principles.md).
4. The operations team places the order, after approval by the chief executive and board of directors, if needed.

Reply:
Agreed :)

Reply:
Hey hey! These changes have not been applied to the file.

Reply:
Great (upside down smiley) -- on it

Reply:
Done